examples/energy_mng/ui/theme.slint

```diff
@@ -28,22 +52,59 @@ export struct Spaces {
 export struct Durations {
     fast: duration,
     medium: duration,
+    slow: duration,
+}
+
+struct Font {
+    size: length,
```

Comment:
So far I see lots of layouts and just very few coded pixel sizes (mostly paddings, etc.) - which is great. Makes me wonder if it would perhaps more sense to code these in relative-font-size here and this prepare it up-front for better scalability. (Just a detail, feel free to ignore :)

Reply:
I like your idea. I plan to make a polish PR at the end also after checked it with the designer. Maybe at that point I can check to use relative-font sizes.